trnsysGUI/GUI.py

```diff
@@ -505,12 +505,7 @@ def toggleConnLabels(self):
         self.centralWidget.setConnLabelVis(self.labelVisState)
 
     def exportHydraulicsDdck(self):
-        statusQuo = self.labelVisState
```

Comment:
@martin-neugebauer  Do you no why this was done? I've taken it out no because the label were showing behind the "hydraulic not connected" dialog box for no obvious reason...